apps/polls/assets/Polls.jsx

```diff
@@ -6,18 +6,9 @@ var Poll = React.createClass({
   getInitialState: function () {
     // FIXME: example data
     return {
-      title: 'Getrennte Eltern: Ist das Wechselmodell die beste Lösung für alle?',
-      options: [{
-        label: 'Ja',
-        count: 22434
-      }, {
-        label: 'Nein',
-        count: 40062
-      }, {
-        label: 'Vielleicht',
-        count: 17627
-      }],
-      ownVote: null,
+      title: this.props.poll.title,
```

Comment:
Disclaimer: this question is more to understand react then a review.

Why are title and choices (at least the labels) state values? from the tutorials i would assume they should stay in the props. Unfortunately the choice.count and choice.ownChoice properties are state as they will change on  a vote :(